server/test/read_authzmodel.go

```diff
@@ -2,18 +2,96 @@ package test
 
 import (
 	"context"
-	"errors"
 	"testing"
 
 	"github.com/openfga/openfga/pkg/id"
 	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/pkg/typesystem"
 	"github.com/openfga/openfga/server/commands"
 	serverErrors "github.com/openfga/openfga/server/errors"
 	"github.com/openfga/openfga/storage"
 	"github.com/stretchr/testify/require"
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
+func TestSuccessfulReadAuthorizationModelQuery(t *testing.T, datastore storage.OpenFGADatastore) {
+	var tests = []struct {
```

Comment:
I may have missed it, but if we don't have a test that writes a model without a version, can we test for that? 

Also, we should inject a row into the models table that has an empty schema_version string and make sure the read returns a v1.0 schema version.

These two tests will make sure we cover our bases.

Reply:
We can't do the first one because of the non null constraint.

Reply:
Test added for the second case. It is Postgres specific.